lib/daru/io/importers/shared/request_log_analyzer_patch.rb

```diff
@@ -0,0 +1,73 @@
+module RequestLogAnalyzerPatch
```

Comment:
When I proposed this module name, I've meant the following: You define ONLY `parse_hash` in that module, and then, in importer (not here in separate method) you do the following:
```ruby
parser = RequestLogAnalyzer::Source::LogParser.new(RequestLogAnalyzer::FileFormat.load(format))
parser.extend(RequestLogAnalyzerPatch) # only this instance has method replaced
parser.parse_hash(path)
```
This way it is easy to see what you "inserting" into the third-party gem, and keep it local.